cypress/e2e/page/categorization-pagetests.cy.js

```diff
@@ -0,0 +1,292 @@
+                cy.loadDataDictionary(p_config.source_folder, p_config.data_dictionary)
+                    .as("dataDictionary");
+            });
+
+            // 2. Open index page
+            // NOTE: Home is visited because some state-related store structures
+            // needed for the categorization page are set up on index page creation
+            cy.visit("/");
+
+            // 3. Load state for categorization page
+
+            // A. Save data table in store
+            cy.get("@config").then(p_config => {
+
+                cy.get("@dataTable").then(p_dataTable => {
+
+                    cy.dispatchToNuxtStore("saveDataTable", {
+
+                        data: p_dataTable,
+                        filename: p_config.data_table,
+                        fileType: "tsv"
+                    });
+                });
+            });
+
+            // B. Enable access to the categorization page
+            cy.dispatchToNuxtStore("initializePage", {
+
+                enable: true,
+                pageName: "categorization"
+            });
+
+            // C. Save data dictionary in store
+            cy.get("@config").then(p_config => {
+
+                cy.get("@dataDictionary").then(p_dataDictionary => {
+
+                    cy.dispatchToNuxtStore("saveDataDictionary", {
+
+                        data: p_dataDictionary,
+                        filename: p_config.data_dictionary,
+                        fileType: "json"
+                    });
+                });
+            });
+
+            // 4. Move to categorization page
+            // NOTE: Routing to the page prevents the Vuex store from being wiped
+            // when a page is 'visited' by Cypress
+            cy.window().its("$nuxt.$router").then(router => {
+
+                router.push({ path: "/categorization" });
+            });
+        });
+
+        // Description of task: User selects one non-tool group category and paints one column
+        // Expected results: Annotation nav and Next page button are enabled
+        it("simple categorization", () => {
+
+            // Action 1
+
+            // Select subject ID category
+            cy.get("[data-cy='categorization-table']")
+                .contains("Subject ID")
+                .click();
+
+            // Assert 1 - Nav and next button should not yet be enabled
+            cy.get("[data-cy='menu-item-annotation'] a")
+                .should("have.class", "disabled");
+            cy.get("[data-cy='button-nextpage']")
+                .should("have.class", "disabled");
+
+            // Action 2
+
+            // Link the first column in the column select table with the subject ID category
+            cy.get("[data-cy='column-linking-table'] tbody tr td")
+                .contains("participant_id")
+                .click();
+
+            // Assert 2 - Nav and next button should be enabled
+            // Assert 1
```

Comment:
I'm following the testing flow of Setup -> Action(s) -> Related assert(s). The commenting scheme I came up with to organize this with a description for the overall action and related asserts. The numbering is to keep track of the nth action/asserts of this individual test.

Reply:
I see. But here you have both `Assert 2` and `Assert 1`, and I am not sure what that refers to. I like the `Assert` comments where the reason behind the expectation isn't obvious from the code. E.g. `Nav and next button should not yet be enabled` is helpful, because I don't know what the app should do after I click on "Subject ID". I would even consider adding: `" because no column has been categorized yet"` or something like that.

On the other hand, I don't find the comments about actions helpful, because they restate what follows in the code (e.g. `"Select subject ID category"`). So I would consider getting rid of them